ApplyToBecomeInternal/ApplyToBecomeCypressTests/cypress/support/commands.js

```diff
@@ -86,15 +86,15 @@ Cypress.Commands.add('submitDateLaInfoSent', (day, month, year) => {
 	cy.get('[id="la-info-template-sent-date-month"]').clear().type(month)
 	cy.get('[id="la-info-template-sent-date-year"]').clear().type(year)
 	cy.saveLocalStorage()
-});
+})
```

Comment:
Did you mean to remove the `;`?

Reply:
Yes, as a writing guide on cypress semicolons aren't necessary. Though I refrained from removing other semicolons as I'll open another PR just to target this.